tests/suite/fixtures.py

```diff
@@ -767,9 +773,9 @@ def transport_server_setup(
     global_config_file = f"{TEST_DATA}/{request.param['example']}/standard/global-configuration.yaml"
     gc_resource = create_gc_from_yaml(kube_apis.custom_objects, global_config_file, "nginx-ingress")
 
-    # deploy dns
-    dns_file = f"{TEST_DATA}/{request.param['example']}/standard/dns.yaml"
-    create_items_from_yaml(kube_apis, dns_file, test_namespace)
+    # deploy service_file
+    service_file = f"{TEST_DATA}/{request.param['example']}/standard/service.yaml"
```

Comment:
`service.yaml` includes a deployment and a service, not just a service. 
because service in Kubernetes has a meaning - it is one of the k8s resources - perhaps we should use a different name here? For example, tcp-app.yaml ? and app_file for the variable.   just to avoid any potential confusion.

Reply:
I was hoping to pick a name that's common across status and load balancing test, so I've updated to `service_deployment.yaml` if that's okay